java/src/test/java/eu/solven/cleanthat/engine/java/refactorer/cases/do_not_format_me/TestUnnecessaryCaseChangeCases.java

```diff
@@ -424,4 +425,8 @@ public Object pre() {
         }
     }
 
+    // This class threw a strange StackOverFlowError with an earlier implementation of the mutator
+    @UnmodifiedCompilationUnitAsResource(pre = "source/do_not_format_me/UnnecessaryCaseChange/SomeClass.java")
```

Comment:
Could you open a PR with this commit, as I underastand it leads to weird behavior.

Reply:
@gbq6 TY